Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -337,11 +337,13 @@ private void TextBox_TextChanged(object sender, TextChangedEventArgs e)
 
             if (FilesystemHelpers.ContainsRestrictedCharacters(textBox.Text))
             {
+                FileNameTeachingTip.Visibility = Visibility.Visible;
                 FileNameTeachingTip.IsOpen = true;
             }
             else
             {
                 FileNameTeachingTip.IsOpen = false;
+                FileNameTeachingTip.Visibility = Visibility.Collapsed;
```

Comment:
What is the purpose of this change? I think setting `IsOpen` to false is enough to hide tip.

Reply:
@tsvietOK If you don't collapse it, it shows up on the tab stack. This change fixes #2919